pkg/v1/mutate/rebase.go

```diff
@@ -46,6 +46,7 @@ func Rebase(orig, oldBase, newBase v1.Image) (v1.Image, error) {
 			return nil, fmt.Errorf("failed to get digest of layer %d of %q: %v", i, orig, err)
 		}
 		if oldLayerDigest != origLayerDigest {
+			// TODO: this is a bad error message...
```

Comment:
What should we do instead?

Reply:
Meh, it's fine. I don't have a better alternative in mind, so I've just removed the comment.